lib/heex_formatter/phases/format.ex

```diff
@@ -154,7 +182,14 @@ defmodule HeexFormatter.Phases.Format do
   end
 
   defp indent_expression(indentation) do
-    String.duplicate(@tab, indentation)
+    String.duplicate(@tab, max(0, indentation))
```

Comment:
Nice ✨ . It avoid the formatter to break when the indentation becomes negative. But it is likely a bug since it will format wrong 🙈

Reply:
Yeah. it's just fighting a symptom, but couldn't figure out, how it becomes negative yet.

Reply:
TBH, I guess it is better to keep as it was so we know that there is something wrong. Or we can raise when it becomes negative with a better error message telling which was the last state. That would help us debug the issue 🤔

Reply:
done